src/devices/tuya.ts

```diff
@@ -6987,6 +6987,87 @@ const definitions: Definition[] = [
         },
         whiteLabel: [tuya.whitelabel('iHseno', 'TY_24G_Sensor_V2', 'Human presence sensor 24G', ['_TZE204_ztqnh5cg'])],
     },
+    {
+        fingerprint: tuya.fingerprint('TS0601', ['_TZE204_laokfqwu']),
+        model: 'WZ-M100',
+        vendor: 'Wenzhi',
+        description: 'Human presence sensor',
+        fromZigbee: [tuya.fz.datapoints],
+        toZigbee: [tuya.tz.datapoints],
+        onEvent: tuya.onEventSetTime,
+        configure: tuya.configureMagicPacket,
+        exposes: [
+            e.illuminance_lux(),
+            e
+                .enum('presence_state', ea.STATE, ['none', 'presence'])
+                .withDescription('presence_state'),
+            e
+                .numeric('target_distance', ea.STATE)
+                .withDescription('Distance to target')
+                .withUnit('m'),
+            e
+                .numeric('sensitivity', ea.STATE_SET)
+                .withValueMin(1)
+                .withValueMax(9)
+                .withValueStep(1)
+                .withDescription('sensitivity of the radar'),
+            e
+                .numeric('near_detection', ea.STATE_SET)
+                .withValueMin(0)
+                .withValueMax(10.0)
+                .withDescription('minimum detection range')
+                .withValueStep(0.1)
+                .withUnit('m'),
+            e
+                .numeric('far_detection', ea.STATE_SET)
+                .withValueMin(0)
+                .withValueMax(10.0)
+                .withDescription('maximum detection range')
+                .withValueStep(0.1)
+                .withUnit('m'),
+            e
+                .numeric('distance', ea.STATE)
+                .withValueMin(0)
+                .withValueMax(10.0)
+                .withDescription('target distance')
+                .withValueStep(0.01)
+                .withUnit('m'),
+            e
+                .numeric('interval_time', ea.STATE_SET)
+                .withValueMin(1)
+                .withValueMax(3600)
+                .withDescription('interval_time')
+                .withValueStep(1)
+                .withUnit('s'),
+            e
+                .numeric('detection_delay', ea.STATE_SET)
+                .withValueMin(0)
+                .withValueMax(10.0)
+                .withValueStep(0.1)
+                .withUnit('s')
+                .withDescription('detection delay'),
+            e
+                .numeric('fading_time', ea.STATE_SET)
+                .withValueMax(1500)
+                .withValueMin(5)
+                .withValueStep(5)
+                .withUnit('s')
+                .withDescription('presence timeout'),
+],
+        meta: {
+            tuyaDatapoints: [
+                [1, 'presence_state', tuya.valueConverterBasic.lookup({'none': tuya.enum(0), 'presence': tuya.enum(1)})],
```

Comment:
```suggestion
                [1, 'presence', tuya.valueConverter.trueFalse1],
```

Can you try with this and `e.presence()` in `exposes`?

Reply:
For me that converter doesn't work properly, @Burki24 @Koenkk please take a look of the external converter that I created, it seem to be working for me. @Burki24 please test it and let me know if it is correctly working for you 2.

const fz = require('zigbee-herdsman-converters/converters/fromZigbee');
const tz = require('zigbee-herdsman-converters/converters/toZigbee');
const exposes = require('zigbee-herdsman-converters/lib/exposes');
const reporting = require('zigbee-herdsman-converters/lib/reporting');
const modernExtend = require('zigbee-herdsman-converters/lib/modernExtend');
const e = exposes.presets;
const ea = exposes.access;
const tuya = require('zigbee-herdsman-converters/lib/tuya');

const definition = {
    fingerprint: tuya.fingerprint('TS0601', ['_TZE204_laokfqwu']),
    model: 'WZ-M100',
    vendor: 'Wenzhi',
    description: 'Human presence sensor',
    fromZigbee: [tuya.fz.datapoints],
    toZigbee: [tuya.tz.datapoints],
    configure: tuya.configureMagicPacket,
    exposes: [
            e.illuminance_lux(),
            e.presence(),
            e
                .numeric('target_distance', ea.STATE)
                .withDescription('Distance to target')
                .withUnit('m'),
            e
                .numeric('sensitivity', ea.STATE_SET)
                .withValueMin(1)
                .withValueMax(9)
                .withValueStep(1)
                .withDescription('sensitivity of the radar'),
            e
                .numeric('minimum_range', ea.STATE_SET)
                .withValueMin(0)
                .withValueMax(10.0)
                .withDescription('minimum detection range')
                .withValueStep(0.1)
                .withUnit('m'),
            e
                .numeric('maximum_range', ea.STATE_SET)
                .withValueMin(0)
                .withValueMax(10.0)
                .withDescription('maximum detection range')
                .withValueStep(0.1)
                .withUnit('m'),
            e
                .numeric('detection_delay', ea.STATE_SET)
                .withValueMin(0)
                .withValueMax(10.0)
                .withValueStep(0.1)
                .withUnit('s')
                .withDescription('detection delay'),
            e
                .numeric('fading_time', ea.STATE_SET)
                .withValueMax(1500)
                .withValueMin(5)
                .withValueStep(5)
                .withUnit('s')
                .withDescription('presence timeout')
        ],
        meta: {
            tuyaDatapoints: [
                [1, 'presence', tuya.valueConverter.trueFalse1],
                [2, 'sensitivity', tuya.valueConverter.raw],
                [3, 'minimum_range', tuya.valueConverter.divideBy100],
                [4, 'maximum_range', tuya.valueConverter.divideBy100],
				[9, 'target_distance', tuya.valueConverter.divideBy100],
				[103, 'illuminance_lux', tuya.valueConverter.raw],
				[105, 'detection_delay', tuya.valueConverter.divideBy10],
				[106, 'fading_time', tuya.valueConverter.divideBy10],
            ],
        },
};

module.exports = definition;

Reply:
@OscarCortesMedina , @Koenkk 

Thanks, I adjusted the converter and testr run perfectly.